definitions/mobile.d.ts

```diff
@@ -526,7 +526,7 @@ declare module Mobile {
 	}
 
 	interface IHouseArrestClient {
-		getAfcClientForAppContainer(applicationIdentifier: string): Mobile.IAfcClient;
+		getAfcClientForAppDocuments(applicationIdentifier: string): Mobile.IAfcClient;
```

Comment:
Can we introduce new method instead of renaming the current one? We'll need both of them